<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Admin Center</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>rsc/starclitz-logo.ico</normaloff>rsc/starclitz-logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidgetMain">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabEntry">
       <attribute name="title">
        <string>Entry</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidgetItemType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabCommodities">
           <attribute name="title">
            <string>Commodities</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item row="1" column="3">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="0">
             <widget class="QPushButton" name="pushButtonCommoditiesAdd">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelCommoditiesSell">
              <property name="text">
               <string>Sell</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelCommoditiesBuy">
              <property name="text">
               <string>Buy</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelCommoditiesName">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" colspan="2">
             <widget class="QLineEdit" name="lineEditCommoditiesName"/>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxCommoditiesBuy">
              <property name="maximum">
               <double>99999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxCommoditiesSell">
              <property name="maximum">
               <double>99999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelDescription">
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2" colspan="2">
             <widget class="QLineEdit" name="lineEditCommoditiesDescription"/>
            </item>
            <item row="5" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabLocations">
           <attribute name="title">
            <string>Locations</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="labelLocationsName">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="pushButtonLocationsAdd">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelLocationsSystem">
              <property name="text">
               <string>System</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="lineEditLocationsMoon"/>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="lineEditLocationsPlanet"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelLocationsPlanet">
              <property name="text">
               <string>Planet</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLineEdit" name="lineEditLocationsSystem"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="labelLocationsDescription">
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="lineEditLocationsStation"/>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="lineEditLocationsName"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelLocationsMoon">
              <property name="text">
               <string>Moon</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelLocationsStation">
              <property name="text">
               <string>Station</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLineEdit" name="lineEditLocationsDescription"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabShips">
           <attribute name="title">
            <string>Ships</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="1" colspan="2">
             <widget class="QLineEdit" name="lineEditShipsManufacturer"/>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="pushButtonShipsAdd">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QLineEdit" name="lineEditShipsModel"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelShipsVariant">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Variant</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelShipsModel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Model</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelShipsDescription">
              <property name="text">
               <string>Description</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelShipsBuy">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Buy</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QLineEdit" name="lineEditShipsVariant"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelShipsManufacturer">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Manufacturer</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="1" colspan="2">
             <widget class="QLineEdit" name="lineEditShipsDescription"/>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxShipsBuy">
              <property name="maximum">
               <double>9999999999.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetCurrent">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEditCommoditiesName</tabstop>
  <tabstop>doubleSpinBoxCommoditiesBuy</tabstop>
  <tabstop>doubleSpinBoxCommoditiesSell</tabstop>
  <tabstop>lineEditCommoditiesDescription</tabstop>
  <tabstop>pushButtonCommoditiesAdd</tabstop>
  <tabstop>lineEditLocationsName</tabstop>
  <tabstop>lineEditLocationsStation</tabstop>
  <tabstop>lineEditLocationsMoon</tabstop>
  <tabstop>lineEditLocationsPlanet</tabstop>
  <tabstop>lineEditLocationsSystem</tabstop>
  <tabstop>lineEditLocationsDescription</tabstop>
  <tabstop>pushButtonLocationsAdd</tabstop>
  <tabstop>lineEditShipsManufacturer</tabstop>
  <tabstop>lineEditShipsModel</tabstop>
  <tabstop>lineEditShipsVariant</tabstop>
  <tabstop>doubleSpinBoxShipsBuy</tabstop>
  <tabstop>lineEditShipsDescription</tabstop>
  <tabstop>pushButtonShipsAdd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
